tests/Tests.AzureAppConfiguration/RefreshTests.cs

```diff
@@ -50,37 +48,59 @@ public class RefreshTests
         [Fact]
         public void RefreshTests_RefreshRegisteredKeysAreLoadedOnStartup_DefaultUseQuery()
         {
-            var mockClient = GetMockConfigurationClient();
+            var serviceCollection = new List<ConfigurationSetting>(_kvCollection);
+            var mockResponse = new Mock<Response>();
+            var mockClient = new Mock<ConfigurationClient>(MockBehavior.Strict, TestHelpers.CreateMockEndpointString());
+
+            Response<ConfigurationSetting> GetTestKey(string k, string l, CancellationToken ct)
```

Comment:
>string k, string l, CancellationToken ct [](start = 54, length = 40)

More descriptive names for the method arguments would make the code easier to read.
(k, l, ct) => (key, label, cancallationToken)